Build/Common.Build.Default.props

```diff
@@ -29,8 +29,9 @@
 
   <!-- Common build settings -->
   <PropertyGroup>
-    <OptimizedBuild />
-    <OptimizedBuild Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='Test'">true</OptimizedBuild>
+    <FreeBuild />
+    <FreeBuild Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='Test'">true</FreeBuild>
```

Comment:
Why not. I'll make "DebugBuild" have the value of negation of the current value of FreeBuild.